healpix_alchemy/tests/test_regions.py

```diff
@@ -0,0 +1,103 @@
+import re
+
+from astropy.coordinates import SkyCoord
+from astropy.table import Table
+from astropy import units as u
+from astropy.utils.data import get_readable_fileobj
+from mocpy import MOC
+import numpy as np
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, ForeignKey
+from sqlalchemy.orm import relationship
+
+from .. import Region, Tile
+
+
+Base = declarative_base()
+
+
+class Field(Region, Base):
+    __tablename__ = 'fields'
+
+    field_id = Column(
+        Integer,
+        primary_key=True)
+
+    tile_class = lambda: FieldTile # noqa
```

Comment:
```suggestion
    tile_class = lambda: FieldTile  # noqa: E731
```

Always specify the warning that is being overridden. (For the same reason that you should always specify the exception that is to be caught).

Also, PEP8 says:

> An inline comment is a comment on the same line as a statement. Inline comments should be separated by at least two spaces from the statement. They should start with a # and a single space.